apps/meteor/client/views/omnichannel/contactInfo/EditContactInfo.tsx

```diff
@@ -118,20 +119,38 @@ const EditContactInfo = ({ contactData, onClose, onCancel }: ContactNewEditProps
 
 	const validateEmailFormat = async (emailValue: string) => {
 		const currentEmails = emails.map(({ address }) => address);
-		const isDuplicated = currentEmails.filter((email) => email === emailValue).length > 1;
 
 		if (!validateEmail(emailValue)) {
 			return t('error-invalid-email-address');
 		}
 
-		return !isDuplicated ? true : t('Email_already_exists');
+		if (currentEmails.filter((email) => email === emailValue).length > 1) {
+			return t('Email_already_exists');
+		}
+
+		const initialEmails = initialValue.emails.map(({ address }) => address);
```

Comment:
why mapping `initialEmails`, if you already have `currentEmails` already?

Reply:
because I need the initial ones to validate and not the current... If I'm editing the contact I want to assure that I will not be blocked if the previous value exist already in DB...